USER/StatsDisplay.js

```diff
@@ -30,16 +34,19 @@ Graphics.prototype.setFontMonofonto14 = function () {
   );
 };
 
+//SECTION: Screen drawing
 function draw() {
   if (drawing) return;
   drawing = true;
   bC.clear();
   if (screenSelected == perkScreen) {
-    buildScreen('USER/StatsDisplay/PERKS/ENABLED');
+    buildScreen('USER/PERKS/ENABLED');
```

Comment:
```suggestion
    buildScreen('USER/StatsDisplay/PERKS/ENABLED');
```
These will all need to be updated to match the current directory structure of `/USER/StatsDisplay/*`

Reply:
@gnargle I fixed all these up myself in testing and the app is working good. So once these are fixed we'll be good to go! Thanks!

Reply:
Resolve with latest commit :)